control/dtime.py

```diff
@@ -126,6 +127,7 @@ def c2d(sysc, Ts, method='zoh', prewarp_frequency=None):
     """
 
     #  Call the sample_system() function to do the work
-    sysd = sample_system(sysc, Ts, method, prewarp_frequency)
+    sysd = sample_system(sysc, Ts,
+        method=method, prewarp_frequency=prewarp_frequency)
```

Comment:
So this has not been caught by the unit tests, right? Should we add a test which calls `c2d` with nonstandard kwargs? Maybe parametrize `test_sample_system_prewarp` with `[plant.sample, c2d, sample_system]`

Reply:
good idea; done.